lib/OrdinaryDiffEqExponentialRK/Project.toml

```diff
@@ -4,30 +4,49 @@ authors = ["ParamThakkar123 <paramthakkar864@gmail.com>"]
 version = "1.0.0"
 
 [deps]
+DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
+ExponentialUtilities = "d4d017d3-3776-5f7e-afef-a10c40355c18"
 FastBroadcast = "7034ab61-46d4-4ed7-9d0f-46aef9175898"
+LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
 MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
 OrdinaryDiffEqCore = "bbf590c4-e513-4bbe-9b18-05decba2e5d8"
-RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
-ExponentialUtilities = "d4d017d3-3776-5f7e-afef-a10c40355c18"
-LinearAlgebra = "37e2e46d-f89d-539d-b4ee-838fcccc9c8e"
-DiffEqBase = "2b5f629d-d688-5b77-993f-72d75c75574e"
-SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
-Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 OrdinaryDiffEqDifferentiation = "4302a76b-040a-498a-8c04-15b101fed76b"
-OrdinaryDiffEqVerner = "79d7bb75-1356-48c1-b8c0-6832512096c2"
 OrdinaryDiffEqSDIRK = "2d112036-d095-4a1e-ab9a-08536f3ecdbf"
+OrdinaryDiffEqVerner = "79d7bb75-1356-48c1-b8c0-6832512096c2"
+RecursiveArrayTools = "731186ca-8d62-57ce-b412-fbd966d074cd"
+Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
+SciMLBase = "0bca4576-84f4-4d90-8ffe-ffa030f20462"
 
 [compat]
+DiffEqBase = "6.152.2"
+DiffEqDevTools = "2.44.4"
+ExponentialUtilities = "1.26.1"
+FastBroadcast = "0.3.5"
+LinearAlgebra = "<0.0.1, 1"
+LinearSolve = "2.32.0"
+MuladdMacro = "0.2.4"
+OrdinaryDiffEqCore = "<0.0.1, 1"
+OrdinaryDiffEqDifferentiation = "<0.0.1, 1"
+OrdinaryDiffEqSDIRK = "<0.0.1, 1"
+OrdinaryDiffEqTsit5 = "<0.0.1, 1"
+OrdinaryDiffEqVerner = "<0.0.1, 1"
```

Comment:
Forgot to mention this on slack, for packages without a registered version it assumes stdlib and puts this in compat

Reply:
That compat actually works fine though since the first version for everything is 1.0.